opensuse-kubectl-docker/Dockerfile

```diff
@@ -19,8 +17,11 @@ RUN true \
     curl \
     wget \
     tar \
-    helm \
     direnv \
+    aaa_base bash-completion docker-bash-completion systemd-bash-completion \
+    && wget -q https://kubernetes-helm.storage.googleapis.com/helm-v2.4.2-linux-amd64.tar.gz -O - | tar xz -C /usr/local/bin --strip-components=1 linux-amd64/helm \
```

Comment:
Yes, that worked, with a minor tweak for CRLF cleanup. There may be a better way to do this in awk, but I think `RS` is the right way.